spec/classes/init_spec.rb

```diff
@@ -237,7 +237,7 @@
       it 'should fail' do
         expect {
           should contain_class('nscd')
-        }.to raise_error(Puppet::Error,/nscd supports osfamilies Debian, RedHat and Suse. Detected osfamily is <unsupported>\./)
+        }.to raise_error(Puppet::Error,/Nscd supports osfamilies Debian, RedHat, Suse and Solaris\. Detected osfamily is <unsupported>\./)
```

Comment:
This is the only (non functional) change to rspec to reflect new error message containint Solaris support.